core/domain/question_fetchers_test.py

```diff
@@ -101,7 +101,8 @@ def test_get_questions_by_ids(self):
     def test_cannot_get_question_from_model_with_invalid_schema_version(self):
         # Delete all question models.
         all_question_models = question_models.QuestionModel.get_all()
-        question_models.QuestionModel.delete_multi(all_question_models)
+        for model in all_question_models:
```

Comment:
Why this change?

Reply:
+1

Reply:
My implementation before making review changes needed this, I have modified it back to use `delete_multi`.